Tasks/GoTool/task.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "334727F4-9495-4F9D-A391-FC621D671474",
+    "name": "GoTool",
+    "friendlyName": "Go Tool Installer",
+    "description": "Finds or Downloads and caches specified version spec of Go and adds it to the PATH.",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "satisfies": ["GO"],
+    "demands": [],
+    "instanceNameFormat": "Use Go $(version)",
+    "inputs": [{
+            "name": "version",
+            "type": "string",
+            "label": "Version",
+            "defaultValue": "1.9.2",
+            "required": true,
+            "helpMarkDown": "Golang tool version to download and install.  Examples: 1.9.2"
```

Comment:
Why is it using Golang here? I think we should be consistent with the naming throughout the labels and messages

Reply:
Done. changed it to Go